Matrix.java

```diff
@@ -1,15 +1,16 @@
 package com.company.hw01;
 
 import java.io.Serializable;
-import java.lang.reflect.Array;
 import java.util.Arrays;
 
-public class Matrix  extends Object implements Serializable {
+public class Matrix extends Object implements Serializable {
     private int[][] matrix;
     private int height;
     private int width;
-    private Matrix existring = null;
-    private boolean sorted = false;
+    //if we create an instance of a class, but do not enter data into it,
+    // that we need to check it, otherwise we output an empty matrix
+    private boolean exist;

```

Comment:
remove exist, when you create Matrix  class it should be fully initialized. We should no be able to create class but do not  enter data